x-pack/legacy/plugins/reporting/export_types/common/execute_job/add_force_now_query_string.ts

```diff
@@ -1,60 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License;
- * you may not use this file except in compliance with the Elastic License.
- */
-
-import url from 'url';
-import { getAbsoluteUrlFactory } from '../../../common/get_absolute_url';
-import { ConditionalHeaders, JobDocPayload, KbnServer } from '../../../types';
-
-function getSavedObjectAbsoluteUrl(job: JobDocPayload, relativeUrl: string, server: KbnServer) {
-  const getAbsoluteUrl: any = getAbsoluteUrlFactory(server);
-
-  const { pathname: path, hash, search } = url.parse(relativeUrl);
-  return getAbsoluteUrl({ basePath: job.basePath, path, hash, search });
-}
-
-export const addForceNowQuerystring = async ({
-  job,
-  conditionalHeaders,
-  logo,
-  server,
-}: {
-  job: JobDocPayload;
-  conditionalHeaders?: ConditionalHeaders;
-  logo?: any;
-  server: KbnServer;
-}) => {
-  // if no URLS then its from PNG which should only have one so put it in the array and process as PDF does
-  if (!job.urls) {
-    if (!job.relativeUrl) {
-      throw new Error(`Unable to generate report. Url is not defined.`);
```

Comment:
This check is moved to `x-pack/legacy/plugins/reporting/export_types/common/execute_job/get_full_urls`